### PR TITLE
change input component

### PR DESCRIPTION
**Pull request by martinmcneela**
Probably best leave this one for ZR as we have had a discussion about it previously.

### CODE REVIEW

**Review by zac-robinson — CHANGES_REQUESTED** [review]
bump the version but otherwise looks good

**Comment by martinmcneela** [comment]
> bump the version but otherwise looks good

Done @zac-robinson

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "mechanical-wombat",
-  "version": "0.17.2",
+  "version": "0.17.3",
```
Comment (zac-robinson) [inline package.json:3]: If this is going in after branding, the version needs to be above 1.0.1

**Review by zac-robinson — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (martinmcneela) [reply package.json:3]: I'm just going to hold it for now then, once things have calmed down i'll update it and do it then.

Comment (martinmcneela) [reply package.json]: right, done it now.

**Review by zac-robinson — APPROVED** [review]

**Pull request merged**